### PR TITLE
pixi loader passthrough

### PR DESCRIPTION
**Pull request by mbittarelli-jibo**
* Adding parameters to the PIXI.animate load method to passthrough to the PIXI.loader
* Adding documentation

### CODE REVIEW

File: src/animate/load.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ const load = function(options, parent, complete, basePath, loader, metadata) {
         complete: null
     }, options || {});
 
-    const loader = new PIXI.loaders.Loader();
+    loader = loader ? loader : new PIXI.loaders.Loader();
```
Comment (bigtimebuddy) [inline src/animate/load.js]: Would prefer `loader = loader || new PIXI.loaders.Loader()`

**Comment by bigtimebuddy** [comment]
Yes I can publish. There are a bunch of other fixes to get in

**Comment by thagberg** [comment]
@bigtimebuddy Can you add "thagberg" as an npm owner?

**Comment by bigtimebuddy** [comment]
@thagberg sure. Could you also add me back as a contributor to the pixi-animate and pixi-animate-extension repos? Looks like I was removed.

**Comment by thagberg** [comment]
Ah, that must have happened when devops added me as an owner today.

**Comment by thagberg** [comment]
@bigtimebuddy you should have an invite with write permissions

**Pull request merged**